mkdocs/docs/configuration.md

```diff
@@ -372,7 +372,73 @@ Specific headers defined by the RESTCatalog spec include:
 
 | Key                                  | Options                               | Default              | Description                                                                                                                                                                                        |
 | ------------------------------------ | ------------------------------------- | -------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| `header.X-Iceberg-Access-Delegation` | `{vended-credentials,remote-signing}` | `vended-credentials` | Signal to the server that the client supports delegated access via a comma-separated list of access mechanisms. The server may choose to supply access via any or none of the requested mechanisms |
+| `header.X-Iceberg-Access-Delegation` | `{vended-credentials,remote-signing}` | `vended-credentials` | Signal to the server that the client supports delegated access via a comma-separated list of access mechanisms. The server may choose to supply access via any or none of the requested mechanisms. When using `vended-credentials`, the server provides temporary credentials to the client. When using `remote-signing`, the server signs requests on behalf of the client. |
+
+#### Authentication Options
```

Comment:
i like this section a lot. I think we should move this above the headers section. 

i would like to break up the table above and move the auth related key/values here